Programs/_bootstrap_python.c

```diff
@@ -0,0 +1,102 @@
+
+/* Frozen modules bootstrap */
+
+/* This file is linked with "bootstrap Python"
+   which is used (only) to run Tools/scripts/deepfreeze.py. */
```

Comment:
I suggest a different description:

"Minimum Python executable used to run Tools/scripts/deepfreeze.py when building Python. Configured with an isolated configuration (ex: ignore Python environment variables)."